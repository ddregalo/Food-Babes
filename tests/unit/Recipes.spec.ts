import { shallowMount } from '@vue/test-utils';
const Recipes = require('../../src/views/Recipes.vue');

describe('Recipes View', () => {
  const wrapper = shallowMount(Recipes);

  it('renders the correct markup', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});