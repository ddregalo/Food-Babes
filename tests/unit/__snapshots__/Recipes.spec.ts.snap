// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipes View renders the correct markup 1`] = `undefined`;
